python/common_modules_cff.py

```diff
@@ -117,6 +117,38 @@
     ) * probeMuonsIsoValueMaps
 )
 
+muonMiniIsoCharged = cms.EDProducer("MuonMiniIso",
+    probes = cms.InputTag("probeMuons"),
+    pfCandidates = cms.InputTag("pfAllChargedHadronsPFBRECO"),
+    dRCandProbeVeto = cms.double(0.0001),
+    CandPtThreshold = cms.double(0.0),
+    CandPdgId = cms.string('h'), # 'h','gamma','h0' or leave empty for all
+)
+
+muonMiniIsoPUCharged = cms.EDProducer("MuonMiniIso",
+    probes = cms.InputTag("probeMuons"),
+    pfCandidates = cms.InputTag("pfPileUpAllChargedParticlesPFBRECO"),
+    dRCandProbeVeto = cms.double(0.0001),
+    CandPtThreshold = cms.double(0.0),
+    CandPdgId = cms.string('h'), # 'h','gamma','h0' or leave empty for all
```

Comment:
I think this one should be empty, for consistency with what is done in the current PF isolation (though in practice it makes no difference since pfPileUp never marks as pileup any pf electron or muon)